src/main.rs

```diff
@@ -104,10 +136,20 @@ async fn main() {
         jpeg,
         resize_icons,
         output_dir,
+        output_format,
     } = command.options();
 
     let resize_icons = resize_icons.map(|r| r.unwrap_or(100));
 
+    let html_output_prefix;
+    let epub_output_prefix;
+    (html_output_prefix, epub_output_prefix) =
+        if matches!(output_format, OutputFormat::Both) || output_dir.is_none() {
+            ("html/", "epub/")
+        } else {
+            ("", "")
+        };
```

Comment:
```suggestion
    let mut html_outpu_dir = output_dir.clone().unwrap_or_else(|| PathBuf::from(DEFAULT_OUTPUT_DIR));
    let mut html_outpu_dir = output_dir.clone().unwrap_or_else(|| PathBuf::from(DEFAULT_OUTPUT_DIR));
    
    if output_format == OutputFormat::Both || output_dir.is_none() {
        html_outpu_dir = html_outpu_dir.join("html/");
        epub_outpu_dir = epub_outpu_dir.join("epub/");
    }
```
Doing `.join(if subfolders { "html/" } else { "" })` might also work